WAW.API/Shared/Persistence/Contexts/AppDbContext.cs

```diff
@@ -10,6 +11,8 @@ public class AppDbContext : DbContext {
   private DbSet<Offer>? offers;
   private DbSet<User>? users;
   private DbSet<Company>? companies;
+  private DbSet<ChatRoom> chatRooms;
+  private DbSet<Message> messages;
```

Comment:
These ones should be nullable as the others.

```suggestion
  private DbSet<ChatRoom>? chatRooms;
  private DbSet<Message>? messages;
```